ci/github-script/reviewers.js

```diff
@@ -0,0 +1,129 @@
+// Caching the list of team members saves API requests when running the bot on the schedule and
+// processing many PRs at once.
+const members = {}
```

Comment:
Actually, I don't think this is used anywhere in the script.

Either I suggest removing it:

```suggestion
```

or make use of it for caching, as suggested by the comment

Reply:
I believe this was added in preparation for this TODO:

https://github.com/wolfgangwalther/nixpkgs/blob/fc69b69871461448b6549962ed9b2a7e5bd6fcd7/ci/github-script/bot.js#L389-L393

However, a better way to handle that may be for `bot.js` to pass `reviewers.js` a reference to its `getTeamMembers` function.